.gitignore

```
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+text.md
```

- [ ] I could observed that your test.md file is not yet ignored by git because there is a typo here, kindly correct it. 👍 
```suggestion
test.md
```